### ISSUE TITLE
RLS fails to build on Darwin

### ISSUE THREAD

**Issue by sheeldotme** [NixOS/nixpkgs#65970]
**Describe the bug**
RLS fails to build as libgit2 references a missing file "Security/SecureTransport.h".

```
cargo:warning=libgit2/src/streams/stransport.c:13:10: fatal error: 'Security/SecureTransport.h' file not found
cargo:warning=#include <Security/SecureTransport.h>
cargo:warning=         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
cargo:warning=1 error generated.
exit code: 1

--- stderr
fatal: not a git repository (or any of the parent directories): .git
thread 'main' panicked at '

Internal error occurred: Command "/nix/store/qjbrzscc9kdajnixyc969h16h8qq060h-clang-wrapper-7.1.0/bin/cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "/private/var/folders/5f/ksjnkgp15qs0c2z_3zq2ffdr0000gn/T/nix-build-rls-1.34.0.drv-0/source/target/x86_64-apple-darwin/release/build/libgit2-sys-4936ab6e41c8aa86/out/include" "-I" "libgit2/src" "-I" "libgit2/deps/http-parser" "-I" "libgit2/deps/regex" "-I" "/private/var/folders/5f/ksjnkgp15qs0c2z_3zq2ffdr0000gn/T/nix-build-rls-1.34.0.drv-0/source/target/x86_64-apple-darwin/release/build/libssh2-sys-2c1937ca8cc5637a/out/include" "-fvisibility=hidden" "-o" "/private/var/folders/5f/ksjnkgp15qs0c2z_3zq2ffdr0000gn/T/nix-build-rls-1.34.0.drv-0/source/target/x86_64-apple-darwin/release/build/libgit2-sys-4936ab6e41c8aa86/out/build/libgit2/src/streams/stransport.o" "-c" "libgit2/src/streams/stransport.c" with args "cc" did not execute successfully (status code exit code: 1).

', /private/var/folders/5f/ksjnkgp15qs0c2z_3zq2ffdr0000gn/T/nix-build-rls-1.34.0.drv-0/rls-1.34.0-vendor/cc/src/lib.rs:2314:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
builder for '/nix/store/fll2202q8gjcf730p4hvvcb3giqk6b8x-rls-1.34.0.drv' failed with exit code 101
```
**To Reproduce**
1. Create a shell with rls as a buildInput

**Expected behavior**
Successful build

**Screenshots**
N/A
**Additional context**
cargo:warning=libgit2/src/streams/stransport.c:13:10: fatal error: 'Security/SecureTransport.h' file not found

**Metadata**
Sheels-MacBook-Pro:cordial-jalepeno sheelpatel$ nix run nixpkgs.nix-info -c nix-info -m
 - system: `"x86_64-darwin"`
 - host os: `Darwin 18.6.0, macOS 10.14.5`
 - multi-user?: `no`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.2.2`
 - channels(sheelpatel): `"nixpkgs-19.09pre183693.baa75f0c25a"`
 - nixpkgs: `/Users/sheelpatel/.nix-defexpr/channels/nixpkgs`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attributes: Security
# a list of nixos modules affected by the problem
module: rls
```

**Comment by matthias-t** [comment]
correct me if I'm wrong, but I believe this was fixed in #63759

**Comment by marsam** [comment]
Fixed in #63759
